FWCore/Services/plugins/Timing.cc

```diff
@@ -440,6 +488,7 @@ namespace edm {
                                  << " - Total job:   " << total_job_time << "\n"
                                  << " - EventSetup Lock: " << accumulatedTimeForLock_ << "\n"
                                  << " - EventSetup Get:  " << accumulatedTimeForGet_ << "\n"
+                                 << " - Total ES:    " << accumulatedEventSetupModuleTimings_.load() << "\n"
```

Comment:
How about
```suggestion
                                 << " - Total EventSetup:    " << accumulatedEventSetupModuleTimings_.load() << "\n"
```
?

I expect this line to create a conflict with https://github.com/cms-sw/cmssw/pull/41023 (trivially resolvable, but still)